app-frontend/src/app/components/scenes/sceneDetailModal/sceneDetailModal.controller.js

```diff
@@ -58,4 +63,90 @@ export default class SceneDetailModalController {
     closeWithData(data) {
         this.close({$value: data});
     }
+
+    toggleMetadataEdit() {
+        this.isEditMetadata = !this.isEditMetadata;
+        if (!this.isEditMetadata) {
+            this.updateMetadata();
+        }
+    }
+
+    setAqcDateDisplay() {
+        return this.scene.filterFields && this.scene.filterFields.acquisitionDate ?
+            this.formatAcqDate(this.scene.filterFields.acquisitionDate) :
+            'MM/DD/YYYY';
+    }
+
+    updateMetadata() {
+        // TODO: visibility and data source should be editable eventually
+        // TODO: sunAzimuth and sunElevation can't be changed, might be backend issue?
```

Comment:
I would remove this TODO about the backend problem and make an issue for it.

Reply:
Will make this issue after this branch is merged.